### DISCUSSION TITLE
Your school is a GitHub Campus partner. Please contact your school administrator to have the correct benefits applied to your GitHub account.

### DISCUSSION THREAD

**Discussion by LucaCuello** [community/community#42165] [GitHub Education]
Hi!
What can I do in this case? 
My school said that this has nothing to do with them. I've sent some emails to Github but now they just replied that they're no longer receiving questions like that so I should ask here. 
Anybody had the same issue? :(

**Comment by samus-aran** [comment]
Hi @LucaCuello  👋 !

It looks like your school issues the coupons to its students and faculty via the GitHub Campus program. This means you’ll have to talk to the GitHub administrator for your school and request that they assist you with your Global Campus benefits.

**Reply by GiovanniBraconi** [reply]
I have the same problem and i already tried to talk with my school administrators but nobody knows anything about it. Is there another way?

**Reply by samus-aran** [reply]
Hi @GiovanniBraconi, some partner schools choose to allow students to use the GitHub verification application or else they have their own portal or process. I would suggest you reach out to your University IT support or perhaps computing science lecturers for more information on how your school verifies its students for GitHub Student Developer Pack.

**Reply by arnavmeduri** [reply]
I have the exact same problem! My school advisor says Github should have prompted me to upload my student ID. It doesn't anymore. I don't know where to go from here.

It appears Github doesn't provide phone or email support for issues like this. Disappointed!

**Reply by cristianstortoni** [reply]
same problem

**Reply by MikeDiethelm** [reply]
same problem

**Comment by cdotwang** [comment]
Same question. Really have no idea......

**Comment by joschi27** [comment]
For our school, we have our own campus instance of github (something like schoolname.github.com). Maybe you need to use that instance for whatever you are trying to do?

**Reply by lesi-nedo** [reply]
It helped
